scripts/src/java/org/oppia/android/scripts/maven/maveninstall/MavenListDependency.kt

```diff
@@ -0,0 +1,19 @@
+package org.oppia.android.scripts.maven.maveninstall
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/**
+ * Data class that contains all the details relevant to a dependency that is present
+ * in maven_install.json.
+ */
+@JsonClass(generateAdapter = true)
+data class MavenListDependency(
+  @Json(name = "coord") val coord: String,
+  @Json(name = "mavenListDependencies") val dependencies: List<String>? = null,
```

Comment:
Is the defaulting to null necessary for this & other attributes?

Reply:
hmm, I saw it wasn't necessary. Fixed.

Reply:
Verfied.